### PR TITLE
Rename unknown and wrongly identified elements

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

File: src/openloco/map/tile.h [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "../station.h"
```
Comment (AaronVanGeffen) [inline src/openloco/map/tile.h]: We really should make a light header just for types like `station_id`.

Comment (IntelOrca) [reply src/openloco/map/tile.h]: I had to do this for Ride in OpenRCT2, you can have something like `station_types.h` which just has forward declarations and typedefs, constants etc.

Comment (AaronVanGeffen) [reply src/openloco/map/tile.h]: We already have a `types.hpp`. Perhaps we should just extend that?

Comment (IntelOrca) [reply src/openloco/map/tile.h]: good point

**Comment by AaronVanGeffen** [comment]
@marijnvdwerf I didn't mean for your PR to do the `station_id` change, but I'm happy you did it all the same. Let's get this in.

**Pull request merged**